source/whats-new.txt

```diff
@@ -53,6 +54,43 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.3:
+
+What's New in 6.3
+-----------------
+
+The {+driver-short+} v6.3 release includes the following features:
+
+- Adds the ``serverMonitoringMode`` client option to control the
+  behavior of the monitoring connection between each node in a topology.
+  This option takes a value of ``auto`` (default), ``poll``, or
+  ``stream``. When the setting is ``auto``, the environment the driver
+  runs in determines the monitoring mode. Function-as-a-service (FaaS)
+  environments prefer the polling mode and all others prefer the
+  streaming mode.
+
+  You can set the ``serverMonitoringMode`` option in a
+  ``MongoClientOptions`` instance or as a connection string option. The
+  following example shows how to create a client with the option set to
+  ``stream``:
+  
+  .. code-block:: js
+     
+     new MongoClient('<connection string>', { serverMonitoringMode: 'stream' });
```

Comment:
S: I would probably leave most or all of this info to be added in the docs ticket for this feature, but it doesn't hurt to leave it in.

Reply:
I can take this out of the whats new and add it to the connection options entry (in this PR as well) instead.